src/node/capturing-browser.ts

```diff
@@ -224,6 +248,8 @@ $doc.body.appendChild($style);
     const mergedScreenshotOptions = mergeScreenshotOptions(baseScreenshotOptions, emittedScreenshotOptions);
     const changed = await this.setViewport(mergedScreenshotOptions);
     if (!changed) return { buffer: null, succeeded: true, variantKeysToPush: [] };
+    await this.setHover(mergedScreenshotOptions);
+    await this.setFocus(mergedScreenshotOptions);
```

Comment:
I expect hover and focus to be in different test cases :dog:
Is it possible to achieve it by applying two UI states?

Reply:
Focusing and hovering are independent of each other. You can apply both state or the part of them.